raiden/log_config.py

```diff
@@ -131,6 +133,8 @@ def configure_logging(
         log_json: bool = False,
         log_file: str = None,
         disable_debug_logfile: bool = False,
+        debug_log_file_name: str = 'raiden-debug.log',
+        _first_party_packages: FrozenSet[str] =_FIRST_PARTY_PACKAGES,
```

Comment:
why are you prefixing this with an underline? This is used by linting tools to indicate the argument is not used.